test/box-luatest/gh_7685_memtx_tree_idx_get_with_nullable_field_phantom_read_test.lua

```diff
@@ -0,0 +1,56 @@
+local server = require('test.luatest_helpers.server')
```

Comment:
Test is passed locally without applying the patch.

Reply:
It's a surpise for me that `get` is translated to `select` request over **IPROTO**, that's why the test does not work as expected: rewrote it using `txn_proxy`.

Reply:
You squashed changes with another commit.

Reply:
Sorry for the hush, fixed.